stable/opsportal/requirements.yaml

```diff
@@ -1,5 +1,5 @@
 dependencies:
   - name: kommander-ui
-    version: "4.0.0"
+    version: "6.22.0"
```

Comment:
6.22.0 was the kommander-ui version you overwrote. If you wanted latest, did you mean 6.31.6?

Reply:
opsportal isnt making it in beta1 anyway (already cut), so this doesnt matter for this PR - we should create a follow-up to bump to the latest tho.